pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -269,7 +276,7 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 		Namespace: r.clusterInfo.Namespace,
 		Monitors:  csi.MonEndpoints(r.clusterInfo.Monitors, cephCluster.Spec.RequireMsgr2()),
 		CephFS: &csi.CsiCephFSSpec{
-			SubvolumeGroup: cephFilesystemSubVolumeGroup.Name,
+			SubvolumeGroup: getSubvolumeGroupName(cephFilesystemSubVolumeGroup),
```

Comment:
@Rakshith-R can you check that updated name only need at this cm for csi?

Reply:
> @Rakshith-R can you check that updated name only need at this cm for csi?

:+1:  I think that covers it.